src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -85,9 +85,8 @@ public class NewAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
 
         [Parameter(
             Mandatory = false,
-            ValueFromPipelineByPropertyName = true,
-            HelpMessage = "Active-Active feature flag")]
-        public bool ActiveActive { get; set; }
```

Comment:
It doesn't matter how many customers are using the cmdlet - you need to obsolete parameters, preferrably several months before deleteing them